>From 347d49628797ddc53b73c4c65fe8de2ecd2993d8 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Mon, 9 Jul 2012 17:23:23 +0200
Subject: [PATCH 1/3] SharedPortEndpoint needs access to the
 Sock::sock_special enum constant

This is a clang compatibility fix.
---
 src/condor_includes/sock.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/condor_includes/sock.h b/src/condor_includes/sock.h
index 5a51c36..ca1b1c8 100644
--- a/src/condor_includes/sock.h
+++ b/src/condor_includes/sock.h
@@ -85,6 +85,7 @@ public:
 	friend class SecMan;
 	friend class SecManStartCommand;
 	friend class SharedPortListener;
+	friend class SharedPortEndpoint;
 
 	/*
 	**	Methods
-- 
1.7.7.6

